packages/recommend-react/src/Recommend.tsx

```diff
@@ -0,0 +1,226 @@
+import { RecommendClient } from '@algolia/recommend';
```

Comment:
modelling the entire batch logic in React means that if you're ever interested in enabling this for regular JS or another framework, you'll need to entirely rewrite the logic.

Reply:
so will you not address this?

Reply:
"Batching" already works in plain JS. We can call `client.getRecommendations` with all the queries. 

For me "batching" in the context of this PR only makes sense if the user is working in a declarative context. So I don't see what can be made more generic / ported to plain JS?